bot.py

```diff
@@ -596,6 +599,32 @@ def _say(self, *, channel_id, text, attachments, message_type):
                 channel_id=repo_info.channel_id,
                 text=f"And also! There is a release already in progress: {release_pr.url}"
             )
+        elif any_new_commits(last_version):
```

Comment:
I get an exception here currently if there's no existing RC release (bootcamp-eng, odl-video-service-eng):

```
python bot_local.py bootcamp-eng release notes
```
```
Release notes since 0.25.0 No new commits is_announcement=False
fatal: bad revision 'v0.25.0..master'
Exception found when handling a message
Traceback (most recent call last):
  File "/Users/mbertrand/code/release-script/bot.py", line 959, in handle_message
    loop=loop
  File "/Users/mbertrand/code/release-script/bot.py", line 931, in run_command
    manager=manager,
  File "/Users/mbertrand/code/release-script/bot.py", line 583, in commits_since_last_release
    elif any_new_commits(last_version):
  File "/Users/mbertrand/code/release-script/release.py", line 177, in any_new_commits
    return int(check_output(["git", "rev-list", "--count", "v{}..master".format(version), "--"])) != 0
  File "/usr/local/Cellar/python/3.6.4_3/Frameworks/Python.framework/Versions/3.6/lib/python3.6/subprocess.py", line 336, in check_output
    **kwargs).stdout
  File "/usr/local/Cellar/python/3.6.4_3/Frameworks/Python.framework/Versions/3.6/lib/python3.6/subprocess.py", line 418, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['git', 'rev-list', '--count', 'v0.25.0..master', '--']' returned non-zero exit status 128.
```

Reply:
Ah, good catch. Should be fixed now.